slides/k8s/owners-and-dependents.md

```diff
@@ -0,0 +1,157 @@
+# Owners and dependents
+
+- Some objects are created by other objects
+
+  (example: pods created by replica sets, themselves created by deployments)
+
+- When an *owner* object is deleted, its *dependents* are deleted
+
+  (this is the default behavior; it can be changed)
+
+- We can delete a dependent directly if we want
+
+  (but generally, the owner will recreate another right away)
```

Comment:
Oooh, lightbulb moment as we've already been seeing this behavior! I really like this explanation.